lib/bolt/pal.rb

```diff
@@ -12,16 +12,15 @@ def initialize(config)
       # Nothing works without initialized this global state. Reinitializing
       # is safe and in practice only happen in tests
       self.class.load_puppet
-      self.class.configure_logging(config[:log_level])
```

Comment:
This was all dead code. Will file a new ticket to restore configuring Puppet's log level when `--debug` or `--verbose` are set.